ToyBox/classes/MonkeyPatchin/Multiclass/LevelUP+Multiclass.cs

```diff
@@ -200,6 +200,9 @@ private static class ApplySpellbookPatch {
                         var spellsKnown = spellbook1.Blueprint.SpellsKnown;
                         var expectedCount = spellsKnown.GetCount(casterLevelAfter, index);
                         var actual = CasterHelpers.GetActualSpellsLearnedForClass(unit, spellbook1, index);
+                        if(spellbook1.Blueprint.Spontaneous) {
+                            actual -= CasterHelpers.CountExternallyAddedSpells(spellbook1, index);
+                        }
```

Comment:
In theory, we shouldn't have to worry about this on non-spontaneous spell books. If that's not the case, let me know. I think doing this for all spell books might cause extra spells to be learned for memorized books, though?

Reply:
No arguments here. Seems good in theory, but theory and practice are totally different things.